data/quarg/quarg missions.txt

```diff
@@ -325,3 +325,150 @@ mission "Quarg Pug Arfecta Warning"
+			choice
+				`	"Hello, I'm <first> <last>. What can you tell me about the Coalition?"`
+				`	"Hello. Is it true that the Heliarchs defeated you and stole your ringworlds?"`
+			`	It continues to stare at you for a while, without as much as blinking. It looks over to a bench in one of the quieter areas here, and gestures for you to follow. As you two sit down it finally speaks up.`
+			`	"Those three species arose in relative rapid succession, all in close proximity, compared to how other species tend to be separated by long periods of time, and longer still their distances. So large are the gaps, that one might never learn of those that came before. But, that was not the case for the three. The first of them, the Saryds, were in many ways much like us Quarg in our younger days, and we perhaps were much too lenient because of that... The Kimek, traumatized by the billions fallen in their famines, sought help to make sure they would never face such a crisis again, and were eager to establish friendly relations with the Saryds. The Arachi, in a way, parallel humanity's development, though they were less keen on letting their inner conflicts be made public. They were the last of the three, and upon meeting their two seniors, invested heavily on research, to catch up.`
+			`	"In the intersecting point of their territories, we built our ringworlds. Seven thousand Earth years ago, the three species of the Coalition sought to take over those rings. After centuries... no, a millennium of fighting, we left that part of the galaxy."`
+			choice
+				`	"Why did they decide to start attacking you and your rings?"`
+				`	"Even now you're more advanced than them, aren't you? How come you lost thousands of years ago, when they were weaker?"`
+					goto lost
+			`	"Whatever reasons they had, whatever drove them to attack us, we cannot quite comprehend..." It stops, not sighing or showing much emotion. It looks at the ceiling again, then back at you. "No. You are tired of hearing this same story from us Quarg, are you not? That the foolish and young species warred for no good reason," it pauses again, contemplating its words. "That war was not the first instance where we lost a ring of ours, but never did any of us wish for any ring to be lost in the first place. With three rings there, with the living space for over 3 trillion Quarg being constructed, we got awfully... protective of that region, in order to to prevent outside interference from causing us trouble again. Our relations with the Kimek and Arachi were a far cry from the initial success and amicability we enjoyed with the Saryds, and when an Arach ship first showed signs of technology that repaired its hull platings mid-flight, an achievement even the Saryds were yet to make, we... jumped to conclusions."`
```

Comment:
```suggestion
			`	"Whatever reasons they had, whatever drove them to attack us, we cannot quite comprehend..." It stops, not sighing or showing much emotion. It looks at the ceiling again, then back at you. "No. You are tired of hearing this same story from us Quarg, are you not? That the foolish and young species warred for no good reason," it pauses again, contemplating its words. "That war was not the first instance where we lost a ring of ours, but never did any of us wish for any ring to be lost in the first place. With three rings there, with the living space for over three trillion Quarg being constructed, we got awfully... protective of that region, in order to to prevent outside interference from causing us trouble again. Our relations with the Kimek and Arachi were a far cry from the initial success and amicability we enjoyed with the Saryds, and when an Arach ship first showed signs of technology that repaired its hull platings mid-flight, an achievement even the Saryds were yet to make, we... jumped to conclusions."`
```